internal/pkg/errors/errors.go

```diff
@@ -194,21 +194,23 @@ func (e *DSAInvalidPlanError) Error() string {
 }
 
 type DatabaseInputFlavorError struct {
-	Service   string
-	Operation string
-	Cmd       *cobra.Command
-	Args      []string
+	Service string
+	Cmd     *cobra.Command
```

Comment:
The `Operation` field was not used for anything?

Reply:
No, I think we forgot to cleanup there. The `Service` field was also not used (expect in the test)